vm/ieleApi.ml

```diff
@@ -116,12 +116,17 @@ let mine_block () =
   let header = {beneficiary=beneficiary;unix_timestamp=timestamp;number=number;difficulty=difficulty;gas_limit=gas_limit} in
   let latest = List.hd !blocks in
   let initial_state = latest.state in
+  let timestamp = Int64.to_string timestamp in
+  if List.length !pendingTx = 0 then begin
+    let new_block = {state=initial_state; timestamp=timestamp} in
+    blocks := new_block :: !blocks
+  end else
```

Comment:
Semicolon has a lower precedence than if. Just running the next line looks wrong, but I don't know how far the else clause should go.

Reply:
Ah, but the `let .. in expr` has an even lower precedence, so all the rest of the lines are part of the `else`.